changelog.json

```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.3.2-beta2": [
+      "[Fixed] Text in error dialogs is now selectable - #7606",
+      "[Fixed] Prevent some stash and move operations from failing silently - #9038",
+      "[Fixed] Removing branches succeeds even if branch is already deleted on remote - #8831",
+      "[Improved] Errors containing raw Git output are now formatted using fixed-width font - #8964",
+      "[Improved] Upgrade to Electron 7 - #8967"
```

Comment:
i know we don't usually note these things, but it seems helpful from a record-keeping debugging standpoint?

Reply:
I don't want to put it in the prod release notes, but if you think there's value in including in the beta ones, I'll leave that up to you to make the call on.

Reply:
yeah definitely not in prod!